tests/renderering/text/Text.test.ts

```diff
@@ -41,6 +43,48 @@ describe('Text', () =>
 
         text.addChild(child);
         text.destroy({ children: true, texture: true });
-        expect(childDestroyOpts).toEqual({ children: true, texture: true, textureSource: true });
+
+        expect(childDestroyOpts).toEqual({ children: true, texture: true });
```

Comment:
The original here should be true

Reply:
Should be cool - Text does not extend Sprite any more - so no texture to destroy